utils/spaceship.getPadNumber/index.js

```diff
@@ -0,0 +1,8 @@
+function getPadNumber(padCount, value) {
```

Comment:
What kind of situation is the util used?🙋‍♂️
I don't know about exactly `PAD` mean.